src/main.rs

```diff
@@ -31,3 +31,185 @@ use std::process::ExitCode;
 fn main() -> ExitCode {
 	cli::init() // Initiate the command line
 }
+
+#[cfg(test)]
+mod cli_integration {
```

Comment:
Shouldn't this be in a top level `tests/` folder where integration tests typically live?

Reply:
Good idea! This means we can drop a separate CI job because integration tests automatically build the binary.